src/integration-test/resources/db/RequestPoolController_poolsAtCourtLocation.sql

```diff
@@ -84,6 +84,12 @@ INSERT INTO juror_mod.juror (juror_number, last_name, first_name, dob, address_l
 ('641700006', 'PERSON6', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
 ('641700007', 'PERSON7', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
 ('641700008', 'PERSON8', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700009', 'PERSON3', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700010', 'PERSON4', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700011', 'PERSON5', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700012', 'PERSON6', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700013', 'PERSON7', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
+('641700014', 'PERSON8', 'TEST', '1990-05-16', '542 STREET NAME', 'ANYTOWN', 'CH1 2AN', true, null),
```

Comment:
Please can you update the last_name values of these new records to make them unique